templates/CRM/common/civicrm.settings.php.template

```diff
@@ -256,18 +256,18 @@ if (!defined('CIVICRM_UF_BASEURL')) {
 /**
  * Specify a Smarty 3 autoload file.
```

Comment:
@eileenmcnaughton should this get updated as well to maybe a Smarty autoload file?

Reply:
@seamuslee001 yes - good catch